doc/examples/config/config-storage/README.md

```diff
@@ -89,10 +89,11 @@ means the operation will be performed for all values.
 #### Put
 
 To put the value by the path use
-`config.storage.put(path, value)`, where:
+`config.storage.put(path, value, opts)`, where:
 
 * `path`: string, path.
 * `value`: string.
+* `opts.ttl`: number, TTL in seconds. By default, keys will not expire.
```

Comment:
I would add information about tarantool version since which it is supported and that the older versions ignore this option.

Reply:
Thanks, updated.